subsys/usb/usb_c/usbc_pe_common.h

```diff
@@ -14,6 +14,12 @@
 #include <zephyr/smf.h>
 #include "usbc_timer.h"
 
+/**
+ * @brief The HardResetCounter is used to retry the Hard Reset whenever there
+ *        is no response from the remote device (see Section 6.6.6)
+ */
+#define N_HARD_RESET_COUNT 2
```

Comment:
Do we have a header file somewhere with a bunch of counters/timeout values, etc. from the specs collected in one place?

Reply:
This was added in a later PR [53275](https://github.com/zephyrproject-rtos/zephyr/pull/53275)